src/gaxios.ts

```diff
@@ -219,21 +232,25 @@ export class Gaxios {
     if (opts.data) {
       if (isStream.readable(opts.data)) {
         opts.body = opts.data;
+      } else if (hasBuffer() && Buffer.isBuffer(opts.data)) {
+        // Do not attempt to JSON.stringify() a Buffer:
+        opts.body = opts.data;
+        if (!hasHeader(opts, 'Content-Type')) {
+          opts.headers['Content-Type'] = 'application/json';
```

Comment:
Why can we assume that a `Buffer` is JSON?  Or is this simply just _less_ broken than it was before?

Reply:
I figured if someone has set no content type header, for our libraries, `application/json` is the best guess, seemed less likely to break people than no content type ...

Having said this, existing people providing a Buffer, would actually have received this:

```
'{"type":"Buffer","data":[104,101,108,108,111]}'
```

so it was completely broken.

Reply:
Great.  Then I'm ok with it 👍